data_steward/deid/aou.py

```diff
@@ -212,7 +213,9 @@ def create_allowed_states_table(input_dataset, credentials):
     """
 
     map_tablename = input_dataset + "._mapping_src_hpos_to_allowed_states"
-    data = pd.read_csv('../deid/config/internal_tables/src_hpos_to_allowed_states.csv')
+    data = pd.read_csv(
+        os.path.join(DEID_PATH, 'config/internal_tables/src_hpos_to_allowed_states.csv')
```

Comment:
This isn't very significant since we use unix based systems for development, but if we were ever to use Windows, the forward slashes would need be removed so something like `os.path.join(DEID_PATH, 'config', 'internal_tables', 'src_hpos_to_allowed_states.csv')` would make it compatible.

Reply:
Nice catch.  Thank you!

Reply:
I force pushed the changes.  So now these comments have an outdated status.